make/targets/openshift/operator/mom.mk

```diff
@@ -0,0 +1,6 @@
+scripts_dir :=$(shell realpath $(dir $(lastword $(MAKEFILE_LIST)))../../../../scripts)
```

Comment:
I think that this line calculates the absolute path to scripts directory that is four levels up from the current Makefile.

Reply:
Correct. Copy-paste from other script :smile: